consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/Vote.java

```diff
@@ -12,39 +12,51 @@
  */
 package tech.pegasys.pantheon.consensus.ibft;
 
+import tech.pegasys.pantheon.consensus.common.VoteType;
 import tech.pegasys.pantheon.ethereum.core.Address;
+import tech.pegasys.pantheon.ethereum.rlp.RLPException;
 import tech.pegasys.pantheon.ethereum.rlp.RLPInput;
 import tech.pegasys.pantheon.ethereum.rlp.RLPOutput;
 
 import com.google.common.base.Objects;
+import com.google.common.collect.ImmutableBiMap;
 
+/**
+ * This class is only used to serialise/deserialise BlockHeaders and should not appear in business
+ * logic.
+ */
 public class Vote {
   private final Address recipient;
-  private final IbftVoteType voteType;
+  private final VoteType voteType;
+
+  public static final ImmutableBiMap<VoteType, Byte> voteToValue =
```

Comment:
This probably should be private with just `ADD_NONCE` and `DROP_NONCE` as public constants.

Reply:
Done.

Reply:
yes. true. done.